CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project are documented in this file.
 Based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.3] - 2023-04-18
+
+### Fixed
+
+- Added ObjectOwnership property to S3 buckets in accordance with new [bucket ACL changes](https://aws.amazon.com/blogs/aws/heads-up-amazon-s3-security-changes-are-coming-in-april-of-2023/)
```

Comment:
Should we add a line here about `aws-sdk` dependency version update?

Reply:
Sure thing!